Helper/Feed/Generator.php

```diff
@@ -49,11 +52,13 @@ public function __construct(
         \Magento\UrlRewrite\Model\UrlFinderInterface $urlFinder,
         \Magento\ConfigurableProduct\Model\ResourceModel\Product\Type\Configurable $configurable,
         \Magento\Framework\App\Helper\Context $context,
-        \Magento\Framework\Url $frontendUrl
+        \Magento\Framework\Url $frontendUrl,
+        \Magento\Framework\Model\ResourceModel\Iterator $iterator
     ) {
         $this->_helperData                = $helperData;
         $this->_storeManager              = $storeManager;
         $this->_productCollectionFactory  = $productCollectionFactory;
+        $this->productFactory             = $productFactory;
```

Comment:
Does this not need to be $this->-productFactory ?

Reply:
No longer required since rework.